src/gz/menu.c

```diff
@@ -481,6 +481,11 @@ void menu_item_disable(struct menu_item *item)
   menu_deselect(item->owner, item);
 }
 
+_Bool menu_item_get_enabled(struct menu_item *item)
```

Comment:
Added this for the "think" state in the checkbox proc. A little uncertain as to whether it's required or not, but from reading other code it looks like the "think" reason is used to refresh the state of the UI? In which case I need some way of querying whether a menu item is enabled or not.